Makefile

```diff
@@ -0,0 +1,51 @@
+DOCKER_REGISTRY ?= gcr.io/linkerd-io
+REPO = $(DOCKER_REGISTRY)/proxy-init
+TESTER_REPO = buoyantio/iptables-tester
+
+.DEFAULT_GOAL := help
+
+.PHONY: help
+help: ## Show this help
+	@echo 'Info: For integration test using KinD, run: make kind-load && make integration-test'
+	@echo 'Info: For integration test using minikube, run: make integration-test'
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
Loved this technique, I'm gonna steal it :slightly_smiling_face:

Reply:
😄 
I got it from here https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html